src/periphery/SablierMerkleLT.sol

```diff
@@ -120,39 +126,59 @@ contract SablierMerkleLT is
                             INTERNAL CONSTANT FUNCTIONS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @dev Calculates the tranches based on the claim amount and the unlock percentages for each tranche.
-    function _calculateTranches(
+    /// @dev Calculates the start time, and the tranches based on the claim amount and the unlock percentages for each
+    /// tranche.
+    function _calculateStartTimeAndTranches(
         uint128 claimAmount
     )
         internal
         view
-        returns (LockupTranched.TrancheWithDuration[] memory tranches)
+        returns (uint40 startTime, LockupTranched.Tranche[] memory tranches)
     {
+        // Calculate the start time.
+        if (STREAM_START_TIME == 0) {
```

Comment:
i consider that it decreases the readability and its more prone to errors

Reply:
I'd like to know why you said that. 

> prone to errors

Build would fail if there is an error in defining if else.

> decreases the readability

its like reading a language "if this then that else that"

but I will leave it as it is.

Reply:
it is a best practice to use brackets:
- https://google.github.io/styleguide/javaguide.html
- https://www.oracle.com/java/technologies/javase/codeconventions-contents.html

Reply:
Fair enough. But since you used "always", I looked at other style guides from Google and found this: https://google.github.io/styleguide/jsguide.html#formatting-braces-all

> Exception: A simple if statement that can fit entirely on a single line with no wrapping (and that doesn’t have an else) may be kept on a single line with no braces when it improves readability. This is the only case in which a control structure may omit braces and newlines.

```javascript
if (shortCondition()) foo();
```

Since we have both `if..else`, this exception does not apply. However, as it mentions that "when it improves readability", I don't see an issue with using `if..else` in line if, of course, it improves readability.